index.html

```diff
@@ -492,6 +498,11 @@ <h1 class="page-header">Red October Management</h1>
 					data.Labels[i] = data.Labels[i].trim();
 					if (data.Labels[i] == "") { data.Labels.splice(i, 1); }
 				}
+				data.Usages = data.Usages.split(',');
+				for(var i=0, l=data.Usages.length; i<l; i++){
+					data.Usages[i] = data.Usages[i].trim();
+					if (data.Usages[i] == "") { data.Usages.splice(i, 1); }
```

Comment:
Re the splicing; I'm not sure if that's a problem. Can someone with more JS experience advise whether and how this should be fixed? I simply copied from the Labels splitting above. Maybe there should be a separate PR to address all the frontend nits